app/client/src/features/dashboard/DashboardEventList.tsx

```diff
@@ -1,85 +1,83 @@
-import { useRouter } from 'next/router';
-import { Button, Card, CardContent, Grid, List, ListItemSecondaryAction, Typography } from '@mui/material';
-import { makeStyles } from '@mui/styles';
+import * as React from 'react';
+import { Grid } from '@mui/material';
+import { isBefore, isAfter } from 'date-fns';
 
-import { DashboardEventListItem } from './DashboardEventListItem';
-import { Event } from './Dashboard';
+import { DashboardEventListDisplay } from './DashboardEventListDisplay';
+import { PreloadedQuery, usePreloadedQuery } from 'react-relay';
+import { DashboardQuery } from '@local/__generated__/DashboardQuery.graphql';
+import { DASHBOARD_QUERY } from './Dashboard';
 
-const useStyles = makeStyles((theme) => ({
-    item: {
-        marginBottom: theme.spacing(4),
-    },
-    card: {
-        padding: theme.spacing(1),
-    },
-    title: {
-        marginBottom: theme.spacing(1),
-    },
-    text: {
-        marginLeft: theme.spacing(1),
-    },
-}));
+export interface Event {
+    node: {
+        id: string;
+        title: string | null;
+        description: string | null;
+        startDateTime: Date | null;
+        endDateTime: Date | null;
+        isViewerModerator: boolean | null;
+        organization: {
+            name: string;
+        } | null;
+    };
+}
```

Comment:
These should all be `readonly` if you want them to match the events node datatype that is being returned by `useDashboardEvents`.